### PR TITLE
dependabot: Updates

### PR DESCRIPTION
**Pull request by phlax**
Commit Message: dependabot: Updates
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

File: examples/grpc-bridge/client/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
 requests>=2.22.0
 grpcio
 grpcio-tools
-protobuf==3.17.2
+protobuf==3.17.3
```
Comment (moderation) [inline examples/grpc-bridge/client/requirements.txt:4]: We are behind on protobuf in the core dependency space. Should this version align with https://github.com/envoyproxy/envoy/blob/main/bazel/repository_locations.bzl#L597-L608 ?

Comment (phlax) [reply examples/grpc-bridge/client/requirements.txt:4]: perhaps, altho im not sure if it matters that much for this client example

do you want me to downgrade it to `3.16.0` ?

Comment (phlax) [reply examples/grpc-bridge/client/requirements.txt:4]: or perhaps we bump the bazel version for it instead

**Comment by phlax** [comment]
seems like there is another update for codeql/action ill update tomorrow

/wait

File: examples/grpc-bridge/client/requirements.txt (hunk 1 above)
Comment (phlax) [reply examples/grpc-bridge/client/requirements.txt:4]: ive bumped the protobuf version pulled in by bazel

Comment (phlax) [reply examples/grpc-bridge/client/requirements.txt:4]: actually im reverting the bazel bump - it requires more than a simple update to the bazel files

also - afaict the protos in this example are built using 

- https://hub.docker.com/layers/grpc/python/latest/images/sha256-6f0898ac290991eab0c262fa18453904f75286f17978d3185164d01400dc042e?context=explore
- https://hub.docker.com/layers/grpc/go/latest/images/sha256-0d3bb1fbfab306680ebaf751992bd2db2a0322106e4b389e85028a027242c2bc?context=explore

these more than 3 years old and use `v3.6.0`

imho we probably still want to bump the client library

**Review by htuch — APPROVED** [review]

**Pull request merged**